perma_web/perma/templates/terms_of_service.html

```diff
@@ -38,15 +38,15 @@ <h1>Perma Terms Of Service</h1>
 
     <div class="col-sm-8 docs reading-body">
 
-      <h2 class="body-ah">Effective Date: September, 23, 2013</h2>
+      <h2 class="body-ah">Effective Date: January, 22, 2019</h2>
```

Comment:
Is this formatting of a date, with a comma between the month and the day, a convention for this kind of doc? I see if was that way before. I'm more used to seeing, `January 22, 2019`

Reply:
That's known among experts as a "dumb typo" :) fixed